src/charm.py

```diff
@@ -220,7 +220,8 @@ def _on_update_status(self, event: EventBase) -> None:
             if not self.health.machine_configured():
                 self._set_status(Status.SYSCONF_NOT_OPTIMAL)
                 return
-        except SnapError:
+        except SnapError as e:
+            logger.error(f"Error: {e}")
```

Comment:
**todo:** I believe that L224 is redundant, as we log out `Status.SNAP_NOT_RUNNING` as a `WARNING` level.

If we really want to keep the actual error (which is usually not that informative), maybe we drop this line to `logger.debug`?

Reply:
While doing the implementation I found it was informative to understand why the SNAP was not starting (because the PID was not found in the end, given the syslog parsing that had to be re-written). 

I can lower the level of logging though!